RoddyCore/src/de/dkfz/roddy/execution/io/fs/BashCommandSet.groovy

```diff
@@ -200,7 +200,7 @@ class BashCommandSet extends ShellCommandSet {
         StringBuilder sb = new StringBuilder("find ")
         sb << "\"" << f.absolutePath << "\""
```

Comment:
To follow the specification of `find` it might be better to include the `-L` flag after find and before `f.absolutePath`